Makefile

```diff
@@ -39,6 +39,29 @@ uninstall: ## Uninstall the environment
 
 .PHONY: build env start stop restart install uninstall
 
+##
+## ----------------------------------------------------------------------------
+##   Proximis
```

Comment:
Your targets may be safer if you put `&&` operators between commands instead of `;`.

Reply:
I think you can also put `&& \` between the second and the third instruction. :-)